skt/reporter.py

```diff
@@ -349,11 +349,20 @@ def getmergeinfo(self):
         return result
 
     def get_kernel_config(self, suffix=None):
-        cfgname = "config.gz" if not suffix else "config_{}.gz".format(suffix)
+        """Add the kernel config to the list of attachments."""
```

Comment:
Could you please document the arguments and the return value, as well, including the behavior when there's no config?

Reply:
Fixed.